apm/monitor.go

```diff
@@ -67,7 +72,9 @@ func (m *basicMonitor) getRecord(id int64) *eventRecord {
 
 	e := m.current[key]
 	if e == nil {
-		e = &eventRecord{}
+		e = &eventRecord{
+			Tags: map[string]int64{},
```

Comment:
Same question as above about zero value map vs empty map.

Reply:
so we can append to it later.